src/App.vue

```diff
@@ -82,20 +82,20 @@ export default {
 				this.$store.commit('setActiveTableId', parseInt(currentRoute.params.tableId))
 				this.setPageTitle(this.$store.getters.activeTable.title)
 				if (!currentRoute.path.includes('/row/')) {
-					this.switchActiveMenuEntry(document.querySelector('header .header-left .app-menu a[title="Tables"]'))
+					this.switchActiveMenuEntry(document.querySelector('header .header-start .app-menu a[title="Tables"]'))
```

Comment:
I think this change also only happened on master/stable30 not older versions so we might need a fallback.